etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,77 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	listenClientUrls := "http://" + hostname + ":2379"
+
+	advClientUrls := "http://" + hostname + ":2379"
+
+	listenPeerUrls := "http://" + hostname + ":2380"
+
+	initialAdvPeerUrls := "http://" + hostname + ":2380"
+
+	etcdStart := exec.Command("/bin/etcd",
```

Comment:
I have created separate issue https://github.com/gluster/glusterd2/issues/70 for this. PR https://github.com/gluster/glusterd2/pull/71 will address this comment.